packages/lisk-cryptography/src/keys.js

```diff
@@ -12,29 +12,33 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import nacl from 'tweetnacl';
+
+import nacl from './nacl';
 import { bufferToHex } from './buffer';
 import { getAddressFromPublicKey } from './convert';
 import hash from './hash';
 
+const { signKeyPair } = nacl;
+
 export const getPrivateAndPublicKeyBytesFromPassphrase = passphrase => {
 	const hashed = hash(passphrase, 'utf8');
-	const { publicKey, secretKey } = nacl.sign.keyPair.fromSeed(hashed);
+	const { publicKeyBytes, privateKeyBytes } = signKeyPair(hashed);
 
 	return {
-		privateKey: secretKey,
-		publicKey,
+		privateKeyBytes,
```

Comment:
Just a remark here as this is a breaking change, but considering the fact that it will be `v2.0`, and it's more clear what the content is, I think this would be a good change.